apps/docs/app/guides/realtime/[[...slug]]/page.tsx

```diff
@@ -15,12 +16,7 @@ const RealtimeGuidePage = async ({ params }: { params: Params }) => {
   return <GuideTemplate {...data!} />
 }
 
-const generateStaticParams =
-  // [Charis 2025-04-21] For some reason, using the IS_PROD export from common
-  // does not work (its type is a function when this is evaluated?)
-  process.env.NEXT_PUBLIC_VERCEL_ENV === 'production'
-    ? genGuidesStaticParams('realtime')
-    : getEmptyArray
+const generateStaticParams = IS_PROD ? genGuidesStaticParams('realtime') : getEmptyArray
```

Comment:
Problem identified and fixed [here](https://github.com/supabase/supabase/pull/35285)